src/vm/hints/secp/field_utils.rs

```diff
@@ -72,18 +66,9 @@ pub fn reduce(
     ids: &HashMap<String, BigInt>,
     hint_ap_tracking: Option<&ApTracking>,
 ) -> Result<(), VirtualMachineError> {
-    let x_reloc = get_relocatable_from_var_name("x", ids, vm, hint_ap_tracking)?;
-
-    let x_d0 = get_integer_from_relocatable_plus_offset(&x_reloc, 0, vm)?;
-    let x_d1 = get_integer_from_relocatable_plus_offset(&x_reloc, 1, vm)?;
-    let x_d2 = get_integer_from_relocatable_plus_offset(&x_reloc, 2, vm)?;
-
-    //SECP_P = 2**256 - 2**32 - 2**9 - 2**8 - 2**7 - 2**6 - 2**4 - 1
-    let sec_p = bigint_str!(
-        b"115792089237316195423570985008687907853269984665640564039457584007908834671663"
-    );
+    let value = pack_from_var_name("x", ids, vm, hint_ap_tracking)?.mod_floor(&*SECP_P);
 
-    let value = pack(x_d0, x_d1, x_d2, &vm.prime).mod_floor(&sec_p);
+    //let value = pack(x_d0, x_d1, x_d2, &vm.prime).mod_floor(&sec_p);
```

Comment:
Try not to leave commented out code.

Reply:
resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/333/commits/ed307c54ee93a1c8994d030108ed33efbba98391